tin/templates/assignments/manage_student.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+
+{% block main %}
+  <h2>Managing Student {{ user.username }}</h2>
```

Comment:
Also, consider showing the default (assignment level) caps on this page, styled in a way that it helps teachers compare